modules/layers/src/geojson-layer/sub-layer-map.js

```diff
@@ -10,6 +10,7 @@ export const POINT_LAYER = {
     props: {
       filled: 'filled',
       stroked: 'stroked',
+      smoothEdges: 'smoothEdges',
```

Comment:
- A prop that only apply to the ScatterplotLayer must be prefixed with `point`
- Update documentation